AndroidPubSub/AuthenticatedPubsub.md

```diff
@@ -0,0 +1,85 @@
+## Using AWS IOT with user sign-in
+
+The sample demonstrates enabling guest users to use the AWS IoT APIs to securely publish-to and subscribe-from MQTT topics. Following steps demonstrates usage of AWS IoT with user sign-in. 
+
+1. Follow the steps [here](https://aws-amplify.github.io/docs/android/authentication) to add authentication UI to your app.
```

Comment:
Change to "First, lets add the sign-in functionality to the app. We provide a drop-in UI that provides a basic set of sign-in screens. You can add that by our default authentication UI to the app by following the steps [here](https://aws-amplify.github.io/docs/android/authentication)"